### PR TITLE
Enforce conventions (First in OFS)

### PR DESCRIPTION
**Pull request by loechel**

### CODE REVIEW

**Comment by loechel** [comment]
@icemac OFS linting is finished.
Please merge, so it will give less conflicts on future work

I will continue linting all of Zope and its subpackages

File: src/OFS/tests/testTraverse.py [hunk 1]
```diff
@@ -188,14 +188,14 @@ def _makeRestricted(self, name='dummy'):
         from OFS.SimpleItem import SimpleItem
 
         class Restricted(SimpleItem):
-            """Instance we'll check with ProtectedMethodSecurityPolicy
-            """
+            """Instance we'll check with ProtectedMethodSecurityPolicy."""
+
             getId__roles__ = None  # ACCESS_PUBLIC
-            def getId(self):
+            def getId(self):  # NOQA: E306  # psydo decorator
```
Comment (icemac) [inline src/OFS/tests/testTraverse.py]: pseudo? (same in next lines)

File: src/OFS/zpt/main.zpt [hunk 2]
```diff
@@ -3,290 +3,235 @@
 <tal:tabs replace="structure here/manage_tabs" />
 
 <main class="container-fluid">
-	<form name="objectItems" method="post"
```
Comment (icemac) [inline src/OFS/zpt/main.zpt]: There are too many changes in this file, I am not able to check whether the code has not changed.

File: src/Testing/ZopeTestCase/__init__.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 # Copyright (c) 2005 Zope Foundation and Contributors.
 #
 # This software is subject to the provisions of the Zope Public License,
-# Version 2.1 (ZPL).  A copy of the ZPL should accompany this distribution.
+# Version 2.1 (ZPL)Testing.ZopeTestCase  A copy of the ZPL should accompany this distribution.
```
Comment (icemac) [inline src/Testing/ZopeTestCase/__init__.py]: I don't think this one was intensional.

File: tox.ini [hunk 4]
```diff
@@ -38,17 +38,16 @@ commands =
 basepython = python2.7
 skip_install = true
 deps =
-    -cconstraints.txt
```
Comment (icemac) [inline tox.ini]: Why now without constraints?

File: tox.ini [hunk 5]
```diff
@@ -38,17 +38,16 @@ commands =
...
 commands =
-    isort --apply --recursive {toxinidir}/src {posargs}
+    isort --apply --recursive {toxinidir}/src setup.py {posargs}
+    #isort --apply --recursive {toxinidir}/src/OFS {posargs}
 
 [testenv:autopep8]
 basepython = python2.7
 skip_install = true
 deps =
-    -cconstraints.txt
```
Comment (icemac) [inline tox.ini]: Dito.

File: src/OFS/zpt/main.zpt (hunk 2 above)
Comment (loechel) [reply src/OFS/zpt/main.zpt]: I did a beautify of the file and arranged the attributes to a readable form.

Might not be easy to automatically compare.

**Pull request merged**